src/main/java/com/metamx/emitter/core/Emitters.java

```diff
@@ -72,6 +72,10 @@ else if (props.getProperty(HTTP_EMITTER_PROP) != null) {
     httpMap.put("recipientBaseUrl", baseUrl);
     httpMap.put("flushMillis", Long.parseLong(props.getProperty("com.metamx.emitter.flushMillis", "60000")));
     httpMap.put("flushCount", Integer.parseInt(props.getProperty("com.metamx.emitter.flushCount", "300")));
+    /**
```

Comment:
hehe, since you're not doing a value check anymore you don't need the warning. If they are not equal it really doesn't matter